pkg/istio-agent/docs/ca.dot

```diff
@@ -0,0 +1,23 @@
+digraph {
```

Comment:
Are these ca.dot and ca.svg autogenerated?

Reply:
ca.dot is made by me, ca.svg is the rendered form for the markdown file

Reply:
basically just the source of the diagram. Its not required to keep it here but may be useful if we need to change it